package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "business-filings-ui",
-  "version": "5.11.5",
+  "version": "5.11.5a",
```

Comment:
This PR will be merged into the build that was in Test earlier today. (Main branch contains other fixes that we don't want to include.)